### PR TITLE
Finally fixed issues with the Mob- and SignSelector in Minecraft 1.13+

### PR DESCRIPTION
**Pull request by juliarn**
To be able to fix this issue, I added a materialName-string to every config-object containing materialIds and deprecated these ids. Users below the Minecraft version 1.13 will still be able to use materialIds for both Sign- and MobSelector, although they are not recommended. Users of Minecraft 1.13 will just have to delete the config and database of Sign- and MobSelector, so they will contain the new materialName property. These config-defaults normally are set in the modules, but because I don't wanted to change the modules, the default of the new materialName property is in the constructor of the config-object itself. It's not the prettiest way, but this could be changed in a future release, when the most users have to updated config files.
I also removed the Spigot1.8.8-Dependency from the cloudAPI and added the 1.13 spigot-api.

### CODE REVIEW

**Comment by juliarn** [comment]
Idk why my last commits that are already merged are here too, I hope that's no problem.

**Comment by GiantTreeLP** [comment]
Please use the version variable in the parent pom.xml

**Comment by juliarn** [comment]
Well, you said I should use the spigot-api in 1.13, the version of the parent pom is for spigot and 1.8.8.

File: cloudnet-api/pom.xml [hunk 1]
```diff
@@ -41,8 +40,8 @@
         </dependency>
         <dependency>
             <groupId>org.spigotmc</groupId>
-            <artifactId>spigot <!-- spigot-api --> </artifactId>
-            <version>${dependency.spigot.version} <!-- 1.13-R0.1-SNAPSHOT --> </version>
+            <artifactId>spigot-api <!-- spigot-api --> </artifactId>
+            <version>1.8.8-R0.1-SNAPSHOT </version>
```
Comment (GiantTreeLP) [inline cloudnet-api/pom.xml]: Why are you using 1.8.8 here?

**Comment by juliarn** [comment]
Okay, I just noticed there is still a little bug with the materials when using the ids

File: cloudnet-api/pom.xml (hunk 1 above)
Comment (juliarn) [reply cloudnet-api/pom.xml]: oh crap, that was for testing purposes

Comment (juliarn) [reply cloudnet-api/pom.xml]: well, both 1.8 and 1.13 would work, but if I use 1.8 I could the getMaterialById Method and would not need reflect

Comment (juliarn) [reply cloudnet-api/pom.xml]: so ok, i'll go with spigot 1.8.8 again

**Pull request closed without merging**